tests/merge.js

```diff
@@ -15,31 +13,31 @@ describe('merge', () => {
     })
     expect(suites.length).toBe(3)
 
-    expect(suites[0].tests.length).toBe(2)
-    expect(suites[0].passes.length).toBe(2)
-    expect(suites[0].failures.length).toBe(0)
+    expect(suites[0].tests.length).toBe(1)
+    expect(suites[0].passes.length).toBe(0)
+    expect(suites[0].failures.length).toBe(1)
```

Comment:
Why the test assertions have changed?

Reply:
the ordering of the files returned is different for some reason, if i run sort on the resulted array of files the old assertions would work, but that's an extra operation for just switching some files around an array

Reply:
thank you, makes sense to leave them unsorted